src/components/UnlockWallet/index.tsx

```diff
@@ -30,6 +31,49 @@ function warnUser(title: string, message?: string | undefined): void {
     message,
   });
 }
+function changeId(chainName : any) {
```

Comment:
This function is declared multiple times in your PR; export and share it?

Reply:
I created `src/utils/changeChain.ts`